app/views/users/edit.html.erb

```diff
@@ -1,3 +1,25 @@
+<%# <form action="<%# users_path %><%#%" method="POST" accept-charset="UTF-8">  %>
+  <%# <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>
+  <%# <label>Username:</label> %>
+  <%# <input type="text" name="user[username]"> %>
+  <%# <label>Password:</label> %>
+  <%# <input type="password" name="user[password]"> %>
+  <%# <label>E-Mail:</label> %>
+  <%# <input type="email" name="user[email]"> %>
+  <%# <input type="submit" value="Submit"> %>
+<%# </form> %>
+
+<%# <%= form_tag("/users", method: "post") do %>
+<%# <%= label_tag(:username) %>
+<%# <%= text_field_tag("user[:username]") %>
+<%# <%= label_tag(:password) %>
+<%# <%= password_field_tag("user[:password]") %>
+<%# <%= label_tag(:email) %>
+<%# <%= email_field_tag("user[:email]") %>
+<%# <%= submit_tag('submit')%>
```

Comment:
:medal_military:

Reply:
Thank You. Can you write your full names so we use in submitting the project completion form